packages/editor-ui/src/views/SettingsCommunityNodesView.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+	<SettingsView>
+		<div :class="$style.container">
+			<div :class="$style.headingContainer">
+				<n8n-heading size="2xlarge">{{ $locale.baseText('settings.communityNodes') }}</n8n-heading>
+				<n8n-button
+					v-if="!isQueueModeEnabled && getInstalledPackages.length > 0 && !isLoading"
+					label="Install"
+					size="large"
+					@click="openNPMPage"
+				/>
+			</div>
+			<div :class="$style.loadingContainer" v-if="isLoading">
+				<n8n-spinner size="large"/>
+				<n8n-text>{{ $locale.baseText('settings.communityNodes.loading.message') }}</n8n-text>
+			</div>
+			<n8n-action-box
+				v-else-if="isQueueModeEnabled"
+				:heading="$locale.baseText('settings.communityNodes.empty.title')"
+				:description="getEmptyStateDescription"
+				:buttonText="$locale.baseText('settings.communityNodes.empty.installPackageLabel')"
+				:calloutText="actionBoxConfig.calloutText"
+				:calloutTheme="actionBoxConfig.calloutTheme"
+				:hideButton="actionBoxConfig.hideButton"
+				@click="openNPMPage"
+			/>
+			<div
+				v-else-if="getInstalledPackages.length === 0"
+				:class="$style.actionBoxContainer"
+			>
+				<n8n-action-box
+					:heading="$locale.baseText('settings.communityNodes.empty.title')"
+					:description="getEmptyStateDescription"
+					:buttonText="$locale.baseText('settings.communityNodes.empty.installPackageLabel')"
+					:calloutText="actionBoxConfig.calloutText"
+					:calloutTheme="actionBoxConfig.calloutTheme"
+					:hideButton="actionBoxConfig.hideButton"
+					@click="openNPMPage"
+				/>
+			</div>
+			<div
+				:class="$style.cardsContainer"
+				v-else
+			>
+				<community-package-card
+					v-for="communityPackage in getInstalledPackages"
+					:key="communityPackage.packageName"
+					:communityPackage="communityPackage"
+				></community-package-card>
+			</div>
+		</div>
+	</SettingsView>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { mapGetters } from 'vuex';
+import SettingsView from './SettingsView.vue';
+import CommunityPackageCard from '../components/CommunityPackageCard.vue';
+
+const PACKAGE_COUNT_THRESHOLD = 31;
+
+export default Vue.extend({
+	name: 'SettingsCommunityNodesView',
+	components: {
+		SettingsView,
+		CommunityPackageCard,
+	},
+	async mounted() {
+		await this.$store.dispatch('communityNodes/fetchInstalledPackages');
```

Comment:
this should be in try/catch as well... this should also show some error notification because if packages are not loaded for some reason, the whole page is basically broken...